rq worker for srm timing out on a vm

# Bug

## Description

The test `TestJournalistApp.test_delete_source_deletes_docs_on_disk` is failing due to longer time
taken by the `srm` command. The current default timeout is 5 seconds. This is a Vagrant vm.

## Steps to Reproduce

$ pytest -v tests/test_journalist.py::TestJournalistApp::test_delete_source_deletes_docs_on_disk

## Expected Behavior

All tests should pass.

## Actual Behavior

```

job = Job(u'7a6bc7e9-e097-4c43-90d4-12997b565355', enqueued_at=datetime.datetime(2017, 10, 18, 15, 18, 55, 908729)), timeout = 5

    def wait_for_redis_worker(job, timeout=5):
        """Raise an error if the Redis job doesn't complete successfully
        before a timeout.
    
        :param rq.job.Job job: A Redis job to wait for.
    
        :param int timeout: Seconds to wait for the job to finish.
    
        :raises: An :exc:`AssertionError`.
        """
        start_time = time.time()
        while time.time() - start_time < timeout:
            if job.result == REDIS_SUCCESS_RETURN_VALUE:
                return
            elif job.result not in (None, REDIS_SUCCESS_RETURN_VALUE):
                assert False, 'Redis worker failed!'
            time.sleep(0.1)
        print(job.result)
>       assert False, 'Redis worker timed out!'
E       AssertionError: Redis worker timed out!
```

## Comments

Raising the timeout for this test on [line 782](https://github.com/freedomofpress/securedrop/blob/develop/securedrop/tests/test_journalist.py#L782) to 20 seconds solves it for now `utils.async.wait_for_redis_worker(job, 20)`

--- a/securedrop/tests/utils/async.py
+++ b/securedrop/tests/utils/async.py
@@ -7,7 +7,7 @@ import time
 REDIS_SUCCESS_RETURN_VALUE = 'success'
 
 
-def wait_for_redis_worker(job, timeout=5):
+def wait_for_redis_worker(job, timeout=60):
     """Raise an error if the Redis job doesn't complete successfully
     before a timeout.
 

